dll/win32/shimgvw/shimgvw.c

```diff
@@ -1420,17 +1475,35 @@ Preview_OnCommand(HWND hwnd, UINT nCommandID)
             break;
 
         case IDC_ROT_CLOCKW:
-            if (g_pImage)
+            if (g_pImage && g_pCurrentFile)
             {
                 GdipImageRotateFlip(g_pImage, Rotate270FlipNone);
                 Preview_UpdateImage(pData);
             }
             break;
 
         case IDC_ROT_COUNCW:
-            if (g_pImage)
+            if (g_pImage && g_pCurrentFile)
```

Comment:
Why does this case (and the one _above_) needs to check for the `g_pCurrentFile` ?
(Note that I however understand why the two new other cases that are below need to check the pointer, though.)

Reply:
Fixed by using `pData->m_szFile`.